modules/gdscript/gd_script.cpp

```diff
@@ -1696,9 +1696,9 @@ void GDScriptLanguage::reload_tool_script(const Ref<Script> &p_script, bool p_so
 //same thing for placeholders
 #ifdef TOOLS_ENABLED
 
-			while (E->get()->placeholders.size()) {
+			for (Set<PlaceHolderScriptInstance *>::Element *P = E->get()->placeholders.front(); P; P = P->next()) {
```

Comment:
I noticed that when I tried to save a GDScript that's a part of a multiscript the editor froze.

This was an endless loop before, this fixes that.